### PR TITLE
feat(ui5-token): implement Horizon themes

### PR DESCRIPTION
**Pull request by niyap**
Related to: #5007

### CODE REVIEW

File: packages/main/src/themes/sap_horizon_hcw/Token-parameters.css [hunk 1]
```diff
@@ -4,4 +4,16 @@
...
+	--_ui5_token_selected_focus_outline: none;
+	--_ui5_token_focus_outline: none;
+	--_ui5_token_focus_offset: 0.0625rem;
+	--_ui5_token_focus_outline_border_radius: 0.25rem;
 }
+
+[data-ui5-compact-size],
+.ui5-content-density-compact,
+.sapUiSizeCompact {
+	--_ui5_token_focus_offset: 0.0625rem;
```
Comment (elenastoyanovaa) [inline packages/main/src/themes/sap_horizon_hcw/Token-parameters.css:18]: You have applied the same offset for cozy and compact. We have a deviation with the values defined in the spec:

"Compact outline-offset: 0.125rem;

Cozy outline-offset: 0.1875rem;"

Should we ask for a spec update?

**Review by elenastoyanovaa — CHANGES_REQUESTED** [review]
- I saw that in the spec, the n-more label is black in regular mode and blue in readonly mode. I checked that both in this PR and in UI5 the n-more label is always blue and bold. Could you please check which is the desired design?

File: packages/main/src/TokenizerPopover.hbs [hunk 2]
```diff
@@ -21,6 +21,7 @@
 		{{/if}}
 		{{#unless hasValueState}}
 			<div class="{{classes.popoverValueState}}" style="{{styles.popoverValueStateMessage}}">
+				<ui5-icon class="ui5-input-value-state-message-icon" name="{{_valueStateMessageIcon}}"></ui5-icon>
```
[the remark this answers is not in the document]
Comment (nikolay-kolarov) [reply packages/main/src/TokenizerPopover.hbs:24]: I agree. Let`s be consistent with UI5.

[the remark this answers is not in the document]
Comment (niyap) [reply packages/main/src/TokenizerPopover.hbs:24]: As discussed with the team, we will leave the icon visible unless we remove the whole value state from the N-more list. For that issue, I will create a BLI

File: packages/main/src/themes/Token.css [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-	border-style: solid;
-	border-color: var(--sapButton_TokenBorderColor);
+	border: var(--sapButton_BorderWidth) solid var(--sapButton_TokenBorderColor);
 	border-radius: var(--_ui5_token_border_radius);
 	color: var(--_ui5_token_text_color);
 	height: var(--_ui5_token_height);
 	box-sizing: border-box;
 	margin-top: var(--_ui5_token_top_margin);
 	margin-bottom: var(--_ui5_token_bottom_margin);
+	margin-right: var(--_ui5_token_right_margin);
```
[the remark this answers is not in the document]
Comment (niyap) [reply packages/main/src/themes/Token.css:11]: That is the the margin between the tokens. It is specified in the Tokenizer specification.

File: packages/main/src/themes/sap_horizon_hcw/Token-parameters.css (hunk 1 above)
Comment (niyap) [reply packages/main/src/themes/sap_horizon_hcw/Token-parameters.css:18]: The specification is updated :)

**Review by elenastoyanovaa — APPROVED** [review]

File: packages/main/src/TokenizerPopover.hbs (hunk 2 above)
[the remark this answers is not in the document]
Comment (niyap) [reply packages/main/src/TokenizerPopover.hbs:24]: > * I saw that in the spec, the n-more label is black in regular mode and blue in readonly mode. I checked that both in this PR and in UI5 the n-more label is always blue and bold. Could you please check which is the desired design?

Done. Fixed with the next patch set :)

**Pull request merged**